### PR TITLE
Export component individually

### PR DESCRIPTION
**Pull request by Trinovantes**
I'll write some tests later when I have more time

### CODE REVIEW

**Comment by joshgoebel** [comment]
Out of curiosity how would we inject into a component?  Just have a function that takes Highlight.js instance as an argument and returns a component using that instance?

File: tsconfig.json [hunk 1]
```diff
@@ -8,10 +8,6 @@
         "allowSyntheticDefaultImports": true,
 
         "outDir": "dist",
-
```
Comment (joshgoebel) [inline tsconfig.json:11]: Curious: What makes this no longer necessary with v11?

Comment (Trinovantes) [reply tsconfig.json:11]: I think v11 fixed the file paths. I couldn't get this to compile once I upgraded to v11 without removing this alias

**Comment by Trinovantes** [comment]
> Out of curiosity how would we inject into a component? Just have a function that takes Highlight.js instance as an argument and returns a component using that instance?

You can pass options when you use the plugin:
```
app.use(hljsVuePlugin, {
  hljs
})
```

Then access it in the plugin install function
```
    install(app, { hljs }) {
        app.component('highlightjs', component) // instead of component object itself, insert a factory function that uses the hljs instance
    },
```

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**